src/components/AppHeader/AppHeader.vue

```diff
@@ -9,64 +9,38 @@
     >
       <img
         src="~/assets/tu-delft-logo.svg"
-        alt="Home"
+        :alt="$t('home')"
       >
-    </NuxtLink>
-
-    <h1 class="app-header__title">
-      {{ $t("title") }}
-    </h1>
-
-    <button
-      ref="filterButton"
-      type="button"
-      class="app-header__button button button--header"
-      @click="$emit('openFilterMenu')"
-    >
-      <SvgIcon
-        name="filter-icon"
-        class="button--header__icon"
-      />
-      {{ $t("filter") }}
 
-      <div
-        v-show="isFiltered"
-        class="app-header__status-indicator"
-      />
-    </button>
+      <h1 class="app-header__logo-text">
+        {{ $t("title") }}
+      </h1>
+    </NuxtLink>
 
     <button
       ref="menuButton"
       type="button"
-      class="app-header__button button button--header"
+      class="button button--navigation button--spacing"
       @click="$emit('openAppMenu')"
     >
       <SvgIcon
         name="menu-icon"
-        class="button--header__icon"
+        class="button--navigation__icon"
       />
       {{ $t("menu") }}
     </button>
   </header>
 </template>
 
 <script setup lang="ts">
-import { storeToRefs } from "pinia";
-import { useSpacesStore } from "~/stores/spaces";
-
 const props = defineProps<{ openedMenu: string | null }>();
 let lastOpenedMenu: string | null = null;
-const spacesStore = useSpacesStore();
-const { isFiltered } = storeToRefs(spacesStore);
 const menuButton = ref(null as null | HTMLButtonElement);
-const filterButton = ref(null as null | HTMLButtonElement);
 watch(
   () => props.openedMenu,
   (newValue) => {
     if (lastOpenedMenu === "app-menu") {
```

Comment:
Now this whole thing with `lastOpenedMenu` seems excessive.
We could omit it altogether and just check that `newValue` is false then focus.
Sincerely, it even works even if we omit the check for false, but I guess that's coincidence.